WebfrontCore/Views/Client/Privileged/Index.cshtml

```diff
@@ -21,4 +23,9 @@
             </div>
         }
     }
-</div>
\ No newline at end of file
+    </div>
+}else{
+    <div class="row">
```

Comment:
This should be in the translation but if we follow my earlier suggestion, then it is no longer needed